components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/config/SpOAuth2ExpiryTimeConfiguration.java

```diff
@@ -1,63 +0,0 @@
-/*
- * Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
- *
- * WSO2 Inc. licenses this file to you under the Apache License,
- * Version 2.0 (the "License"); you may not use this file except
- * in compliance with the License.
- * You may obtain a copy of the License at
- *
- * http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing,
- * software distributed under the License is distributed on an
- * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
- * KIND, either express or implied. See the License for the
- * specific language governing permissions and limitations
- * under the License.
- */
-
-package org.wso2.carbon.identity.oauth2.config;
-
-/**
- *  Runtime representation of Service Provider specific OAuth2 token expiry times.
- */
-public class SpOAuth2ExpiryTimeConfiguration {
```

Comment:
Don't remove any class, if no longer used, make them deprecated until the next major version update